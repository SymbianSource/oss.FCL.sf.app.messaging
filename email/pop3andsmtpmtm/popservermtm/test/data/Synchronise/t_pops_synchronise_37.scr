+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 20 30387
+OK 20 messages
1 1519
2 1558
3 1599
4 1559
5 1355
6 1519
7 1519
8 1598
9 1557
10 1598
11 1519
12 1597
13 1558
14 1312
15 1476
16 1477
17 1557
18 1516
19 1517
20 1477
.
+OK 20 messages
1 t_pops_synchronise:message026
2 t_pops_synchronise:message027
3 t_pops_synchronise:message028
4 t_pops_synchronise:message029
5 t_pops_synchronise:message030
6 t_pops_synchronise:message031
7 t_pops_synchronise:message032
8 t_pops_synchronise:message033
9 t_pops_synchronise:message034
10 t_pops_synchronise:message035
11 t_pops_synchronise:message036
12 t_pops_synchronise:message038
13 t_pops_synchronise:message039
14 t_pops_synchronise:message040
15 t_pops_synchronise:message041
16 t_pops_synchronise:message042
17 t_pops_synchronise:message043
18 t_pops_synchronise:message044
19 t_pops_synchronise:message045
20 t_pops_synchronise:message046
.
-ERR Unknown Command "CAPA"
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message041; Wed, 15 Aug 2001 09:41:00 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2124983; 15 Aug 2001 9:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac1001025562387687@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:41:27 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4N>; Wed, 15 Aug 2001 10:43:45 +0100
Message-ID: <t_pops_synchronise_message041@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 41
Date: Wed, 15 Aug 2001 10:35:54 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-One Forty-One Forty-One Forty-One Forty-One Forty-One Forty-One
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message040; Wed, 15 Aug 2001 09:45:06 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1007624; 15 Aug 2001 9:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac1001025562382692@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:41:07 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4M>; Wed, 15 Aug 2001 10:43:30 +0100
Message-ID: <t_pops_synchronise_message040@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 40
Date: Wed, 15 Aug 2001 10:35:39 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty Forty Forty Forty Forty Forty Forty Forty Forty Forty 
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message039; Wed, 15 Aug 2001 09:56:39 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ab1007624; 15 Aug 2001 9:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac1001025562375289@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:40:12 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB42>; Wed, 15 Aug 2001 10:43:09 +0100
Message-ID: <t_pops_synchronise_message039@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 39
Date: Wed, 15 Aug 2001 10:35:18 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Nine Thirty-Nine Thirty-Nine Thirty-Nine Thirty-Nine Thirty-Nine
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message038; Wed, 15 Aug 2001 09:40:47 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1127261; 15 Aug 2001 9:38 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102556236c04c@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:39:35 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4G>; Wed, 15 Aug 2001 10:42:47 +0100
Message-ID: <t_pops_synchronise_message038@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 38
Date: Wed, 15 Aug 2001 10:34:55 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Eight Thirty-Eight Thirty-Eight Thirty-Eight Thirty-Eight
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message036; Tue, 14 Aug 2001 15:57:09 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ac2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e626e90@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:48:49 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALT>; Tue, 14 Aug 2001 16:50:59 +0100
Message-ID: <t_pops_synchronise_message036@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 36
Date: Tue, 14 Aug 2001 16:43:11 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Six Thirty-Six Thirty-Six Thirty-Six Thirty-Six Thirty-Six Thirty-Six
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message035; Tue, 14 Aug 2001 15:56:54 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ag2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e621e0f@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:48:28 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALR>; Tue, 14 Aug 2001 16:50:19 +0100
Message-ID: <t_pops_synchronise_message035@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 35
Date: Tue, 14 Aug 2001 16:42:31 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Five Thirty-Five Thirty-Five Thirty-Five Thirty-Five Thirty-Five
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message034; Tue, 14 Aug 2001 17:10:21 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ah2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e61d473@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:48:09 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALP>; Tue, 14 Aug 2001 16:49:55 +0100
Message-ID: <t_pops_synchronise_message034@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 34
Date: Tue, 14 Aug 2001 16:42:07 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Four Thirty-Four Thirty-Four Thirty-Four Thirty-Four Thirty-Four
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message033; Tue, 14 Aug 2001 16:18:19 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ak2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e6146ca@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:47:33 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAL3>; Tue, 14 Aug 2001 16:49:38 +0100
Message-ID: <t_pops_synchronise_message033@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 33
Date: Tue, 14 Aug 2001 16:41:50 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Three Thirty-Three Thirty-Three Thirty-Three Thirty-Three
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message032; Tue, 14 Aug 2001 17:02:44 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ad2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e60ff12@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:47:15 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALN>; Tue, 14 Aug 2001 16:49:15 +0100
Message-ID: <t_pops_synchronise_message032@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 32
Date: Tue, 14 Aug 2001 16:41:27 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Two Thirty-Two Thirty-Two Thirty-Two Thirty-Two Thirty-Two Thirty-Two
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message031; Tue, 14 Aug 2001 16:12:55 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ae2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e606802@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:46:36 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALL>; Tue, 14 Aug 2001 16:48:57 +0100
Message-ID: <t_pops_synchronise_message031@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 31
Date: Tue, 14 Aug 2001 16:41:09 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-One Thirty-One Thirty-One Thirty-One Thirty-One Thirty-One Thirty-One
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message030; Tue, 14 Aug 2001 16:46:34 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ab2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5f7dd1@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:45:36 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALG>; Tue, 14 Aug 2001 16:48:40 +0100
Message-ID: <t_pops_synchronise_message030@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 30
Date: Tue, 14 Aug 2001 16:40:52 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty Thirty Thirty Thirty Thirty Thirty Thirty Thirty Thirty Thirty 
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message029; Tue, 14 Aug 2001 16:07:40 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ai2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5dcbbc@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:43:45 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALA>; Tue, 14 Aug 2001 16:46:31 +0100
Message-ID: <t_pops_synchronise_message029@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 29
Date: Tue, 14 Aug 2001 16:38:42 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Nine Twenty-Nine Twenty-Nine Twenty-Nine Twenty-Nine Twenty-Nine
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message028; Tue, 14 Aug 2001 17:13:53 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id al2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5d8482@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:43:27 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK0>; Tue, 14 Aug 2001 16:46:14 +0100
Message-ID: <t_pops_synchronise_message028@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 28
Date: Tue, 14 Aug 2001 16:38:26 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Eight Twenty-Eight Twenty-Eight Twenty-Eight Twenty-Eight
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message027; Tue, 14 Aug 2001 16:42:59 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id am2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5d375c@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:43:07 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK9>; Tue, 14 Aug 2001 16:45:58 +0100
Message-ID: <t_pops_synchronise_message027@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 27
Date: Tue, 14 Aug 2001 16:38:08 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Seven Twenty-Seven Twenty-Seven Twenty-Seven Twenty-Seven
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message026; Tue, 14 Aug 2001 16:10:55 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id af2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5cedd0@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:42:48 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK8>; Tue, 14 Aug 2001 16:45:35 +0100
Message-ID: <t_pops_synchronise_message026@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 26
Date: Tue, 14 Aug 2001 16:37:47 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six
.
** Read cancelled
+OK 
