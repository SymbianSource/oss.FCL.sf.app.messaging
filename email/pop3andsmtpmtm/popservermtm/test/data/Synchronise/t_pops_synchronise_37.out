STAT
LIST
UIDL
CAPA
TOP 15 1
TOP 14 1
TOP 13 1
TOP 12 1
TOP 11 1
TOP 10 1
TOP 9 1
TOP 8 1
TOP 7 1
TOP 6 1
TOP 5 1
TOP 4 1
TOP 3 1
TOP 2 1
TOP 1 1
QUIT
